### DISCUSSION TITLE
Does NextJS compiler emotion config need `importMap`?

### DISCUSSION THREAD

**Discussion by therealgilles** [ben-rogerson/twin.macro#860] [Q&A]
I'm wondering if using NextJS (v14) emotion compiler option requires (or would benefit from) configuring `importMap`:
https://emotion.sh/docs/@emotion/babel-plugin#importmap

twin.macro re-exports emotion exports, so it seems that it would be necessary and/or advantageous.

**Comment by ben-rogerson** [comment]
Heya Giles
I'm not sure what you mean, is there an import issue in Next 14?

**Reply by therealgilles** [reply]
This is related to this SSR/CSR issue: https://github.com/emotion-js/emotion/issues/3176

I was trying to find a way to workaround the issue by modifying the [labelFormat](https://emotion.sh/docs/@emotion/babel-plugin#labelformat), but could not make it work. I then stumbled upon the documentation for `importMap` and as twin.macro re-exports emotion exports, I wondered if the options were not working as expected because of the lack of it.

Thoughts?
